packages/components/src/Badge/BadgeComposition/BadgeSeparator/BadgeSeparator.component.test.js

```diff
@@ -1,23 +1,20 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import { render } from '@testing-library/react';
 import BadgeSeparator from './BadgeSeparator.component';
 
 describe('BadgeSeparator', () => {
 	it('should default render', () => {
-		// given nothing
 		// when
-		const wrapper = mount(<BadgeSeparator />);
+		render(<BadgeSeparator />);
 		// then
-		expect(wrapper.html()).toMatchSnapshot();
+		expect(document.querySelector('.tc-badge-separator')).toBeInTheDocument();
```

Comment:
Nearly same comment than for BadgeIcon : we check style here, which is not RTL's philsophy. Is it relevant to keep them ?